share/spice/playing_cards/content.handlebars

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
Remove the `content: ...` you don't need to specify any content (template to be used) because we're letting the built-in `movies` template do the work for us :)